packages/protocol/scripts/foundry/signature/Sign.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import { Script } from "forge-std/Script.sol";
+import { console2 as console } from "forge-std/console2.sol";
+
+contract Sign is Script {
```

Comment:
Could this be replaced by using [`cast wallet sign`](https://getfoundry.sh/cast/reference/wallet/sign#cast-wallet-sign)?

Reply:
And a more general question I guess: is either of these scripts (Recover.sol, Sign.sol) actually used anywhere?